src/components/AppsPage/index.jsx

```diff
@@ -260,8 +323,68 @@ class AppsPage extends React.Component {
                   }}
                 />
 
+                <div className="PrivateImageCheckField">
+                  <Checkbox
+                    isBlack
+                    onClick={this.togglePrivateImage}
+                    isChecked={isPrivateImage}
+                  />
+                  &nbsp; Private Image
+                </div>
+
+                {isPrivateImage && (
+                  <div className="PrivateImageTabContainer">
+                    <Tabs>
+                      <div index={1} label={<DockerLogo />}>
+                        <div className="PrivateImageInputs">
+                          <BlackInputText
+                            required
+                            placeholder="Docker Username"
+                            name="username"
+                            value={username}
+                            onChange={(e) => {
+                              this.handleDockerCredentialsChange(e);
+                            }}
+                          />
+
+                          <BlackInputText
+                            required
+                            placeholder="Docker Email"
+                            name="email"
+                            value={email}
+                            onChange={(e) => {
+                              this.handleDockerCredentialsChange(e);
+                            }}
+                          />
+
+                          <BlackInputText
+                            required
+                            placeholder="Password"
```

Comment:
@StephenAraka shouldn't this placeholder be **Docker Password**

Reply:
Alright... let me change that.